assets/src/dashboard/app/api/useCategoriesApi.js

```diff
@@ -53,8 +53,6 @@ export default function useCategoriesApi(dataAdapter, { wpApi }) {
         )
       );
     } catch (e) {
-      // eslint-disable-next-line no-console
-      console.error(e);
```

Comment:
We no longer need to send errors to console since we are setting empty collections when the fetch request fails.